docs/start.rst

```diff
@@ -104,22 +104,28 @@ data objects are referenced by *id*, *slug*, and *name*.
 
 	``name`` is an arbitrary, **non unique name** of an object.
 
-Let's say we now want to find some genomes. We don't always know the *id*,
+Let's say we now want to find some genome indices. We don't always know the *id*,
 *slug*, or *name* by heart, but we can use `filters`_ to find them. We will
-query for all genome data objects:
+first count all genome index data objects:
 
 .. _filters: http://resdk.readthedocs.io/en/latest/ref.html#resdk.ResolweQuery
 
 .. literalinclude:: files/start.py
   :lines: 13
 
-There are several unrelated data objects that pass through our filter. We can
-filter the data even further, this time trying to find a genome in a FASTA
-format:
+This is quite a lot of objects! We can filter even further:
 
 .. literalinclude:: files/start.py
   :lines: 15
 
+.. note::
+
+  For a complete list of filtering options use a "wrong" filtering
+  argument and you will receive an informative message with all options
+  listed. For example::
+
+    res.data.filter(foo="bar")
+
 For future work we want to get the genome with a specific slug. We will `get`_
 it and store a reference to it for later:
 
```

Comment:
Below under the "Run alignment," we still use HISAT2. I would recommend that we stick to the STAR in all cases. Please correct.